### PR TITLE
Creation of the eventDetails and addition of staffing button

### PR DESCRIPTION
**Pull request by marinPh**
A PR that will allow users to interact with events by seeing the details and be able to apply to staff

### CODE REVIEW

File: app/src/main/java/com/swent/assos/ui/screens/assoDetails/EventDetails.kt [hunk 1]
```diff
@@ -1,6 +1,158 @@
...
+      showing = confirming)
+
+  Scaffold(
+      modifier = Modifier.testTag("EventDetails").semantics { contentDescription = "EventDetails" },
+      topBar = { TopNewsBar(asso, navigationActions, event) },
+      floatingActionButton = {
+        JoinUsButton {
+          // call the function to apply for staffing
+          confirming = true
+        }
```
Comment (Tikipiou) [inline app/src/main/java/com/swent/assos/ui/screens/assoDetails/EventDetails.kt:84]: I think "Join us" is not a good label for this button

File: app/src/androidTest/java/com/swent/assos/AssoDetailsTest.kt [hunk 2]
```diff
@@ -24,6 +24,7 @@ class AssoDetailsTest : SuperTest() {
     composeTestRule.activity.setContent {
       AssoDetails(assoId = assoId, navigationActions = mockNavActions)
     }
+    Thread.sleep(10000)
```
[the remark this answers is not in the document]
Comment (marinPh) [reply app/src/androidTest/java/com/swent/assos/AssoDetailsTest.kt:27]: For debugging, but forgot to take it away

File: app/src/androidTest/java/com/swent/assos/EventDetailsTest.kt [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  val assoID = "02s16UZba2Bsx5opTcQb"
+  val event = Event("123456", "description", assoID, Uri.EMPTY, "assoId")
+
+  override fun setup() {
+    FirebaseFirestore.getInstance()
+        .collection("events")
+        .document(event.id)
+        .set(event)
+        .addOnSuccessListener {
+          super.setup()
```
[the remark this answers is not in the document]
Comment (marinPh) [reply app/src/androidTest/java/com/swent/assos/EventDetailsTest.kt:29]: You are write

**Review by Tikipiou — APPROVED** [review]
Perfect!

**Pull request merged**